apps/chat/src/types/applications.ts

```diff
@@ -20,7 +20,7 @@ export interface CreateApplicationModel {
   features?: ApplicationFeatures;
   input_attachment_types?: string[];
   max_input_attachments?: number;
-  defaults?: Record<string, unknown>;
+  defaults?: object;
```

Comment:
if second param of generic is really unknown, previous one was definitely better. lets revert this line